tests/parsing_tests.rs

```diff
@@ -133,3 +133,22 @@ fn parse_setup() -> Result<(), ParseError> {
 
     Ok(())
 }
+
+#[cfg(feature = "xinput")]
+#[test]
+fn parse_xi_get_property_reply_format_0() -> Result<(), ParseError> {
+    let mut s = Vec::new();
+
+    s.push(1); // response_type
+    s.push(0); // pad
+    s.extend(&0u16.to_ne_bytes()); // sequence
+    s.extend(&0u32.to_ne_bytes()); // length
+    s.extend(&0u32.to_ne_bytes()); // type
+    s.extend(&0u32.to_ne_bytes()); // bytes_after
+    s.extend(&0u32.to_ne_bytes()); // num_items
+    s.push(0); // format
+    s.extend(&[0; 11]); // pad
+
+    x11rb::protocol::xinput::XIGetPropertyReply::try_parse(&s)?;
```

Comment:
Could you `assert_eq!(Ok(XIGetPropertyReply { all the fields }), XIGetPropertyReply::try_parse(&s));`? Actually checking that it parses as expected should make it slightly clearer what this tests (and would actually test a bit more).
(And yes, the above is best done in more than one line. You get the idea.)